packages/snaps-utils/src/eval.ts

```diff
@@ -39,10 +47,15 @@ export async function evalBundle(bundlePath: string): Promise<EvalOutput> {
 
     let stdout = '';
     let stderr = '';
+    let exports: string[] = [];
 
     assert(worker.stdout, '`stdout` should be defined.');
     assert(worker.stderr, '`stderr` should be defined.');
 
+    worker.on('message', (message: EvalWorkerMessage) => {
+      exports = message.data.exports;
```

Comment:
Maybe check that this is actually the message we are looking for?

Reply:
It's the only message ever sent from the worker, but I can add an assertion if you prefer.

Reply:
Yeah, just in case.